js/app.js

```diff
@@ -0,0 +1,15 @@
+function runGame (name, lowId, highId, divId) {
+  var game = name;
```

Comment:
Rather than use `name` as the first argument you've passed in to this method and then assigning it to `game`, it might make more sense to simply call `name` `game` when you declare the function:

```javascript
function runGame (game, lowId, highId, divId) {
  var lowNum = parseInt(/* ... */);
  var highNum = parseInt(/* ... */);
  game.getNums(lowNum, highNum);
  game.renderFB(divId);
}
```